presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/db/DbResourceGroupConfigurationManager.java

```diff
@@ -116,9 +118,8 @@ public DbResourceGroupConfigurationManager(ClusterMemoryPoolManager memoryPoolMa
     @Override
     protected List<ResourceGroupSpec> getRootGroups()
     {
-        if (lastRefresh.get() == 0) {
-            throw new PrestoException(CONFIGURATION_UNAVAILABLE, "Root groups cannot be fetched from database");
-        }
+        checkMaxRefreshInterval();
```

Comment:
This function will print cannot fetch *selectors* but this should be about *root groups*. They are likely to fail at the same time so I'm ok with printing the same message for both, but it should not print *selectors* in that case.